recipes/libx264/all/conanfile.py

```diff
@@ -118,15 +121,21 @@ def _configure_autotools(self):
                 args.append("--cross-prefix={}".format("{}/bin/{}-linux-{}-".format(ndk_root, arch, abi)))
         if self._is_msvc:
             self._override_env["CC"] = "cl -nologo"
-            self._autotools.flags.append("-{}".format(msvc_runtime_flag(self)))
```

Comment:
but it removes` msvc_runtime_flag` 👀

Reply:
No, it's part of `autotools.flags`. But for some reason `autotools.flags` is ignored in `libx264` for msvc, so it's enforced by passing `autotools.flags` to custom `--extra-cflags`

Reply:
No more warnings in test package:
Release/MD/static: https://c3i.jfrog.io/c3i/misc/logs/pr/9740/3-configs/windows-visual_studio/libx264/20190605//1d690fdf83d82fb1eeea5656f775e7981b344e5e-test.txt
Debug/MDd/static: https://c3i.jfrog.io/c3i/misc/logs/pr/9740/3-configs/windows-visual_studio/libx264/20190605//e03f63a768ca59a76d565ff28923e7805edec16d-test.txt
Release/MT/static: https://c3i.jfrog.io/c3i/misc/logs/pr/9740/3-configs/windows-visual_studio/libx264/20190605//d71fb572b9459fd49e7701a3a14b583c8e727284-test.txt
Debug/MTd/static: https://c3i.jfrog.io/c3i/misc/logs/pr/9740/3-configs/windows-visual_studio/libx264/20190605//9ce76c0d155c0dd54d5fe2439c92645360a94989-test.txt